main.py

```diff
@@ -356,6 +362,53 @@ def __init__(self, *args):
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
 
+		if not "search_for_updates" in self.settings.settings or not "buildId" in self.settings.settings:
+			self.settings.settings["search_for_updates"] = False
+			self.settings.settings["buildId"] = ""
+			self.settings.save()
+
+		if "search_for_updates" in self.settings.settings:
+			if self.settings.settings["search_for_updates"] == True:
+				if os.name == "nt":
+					projectJson = json.loads(requests.get("https://ci.appveyor.com/api/projects/mborgerson/xqemu-c5j6o/branch/master").text)
```

Comment:
What happens if AppVeyor is down / firewall is active, but update-checking is enabled?
Will XQEMU-Manager just hang and crash then?

Related: Is there a rate limit for anonymous API access? Maybe users get banned if they do more than 10 requests per day or something - so just making sure we understand the "rules".